server/twitter/methods.js

```diff
@@ -29,7 +29,7 @@ export const tweetLearning = (learning)=>{
 
   if(Meteor.settings.isModeProduction){
 
-    const status = learning.title.truncate(250) + ' ~' + learning.username + ' #TodayILearned' ;
+    const status = learning.title.replace('@', ' ').truncate(250) + ' ~' + learning.username + ' #TodayILearned' ;
```

Comment:
What is this line for?

Reply:
To avoid ppl to spam other twitter handlers in the learning tweet.